src/components/Person/Person.tsx

```diff
@@ -13,35 +13,80 @@ import { orcidFromUrl } from "../../utils/helpers"
 import Pager from '../Pager/Pager'
 // import { count } from 'console'
 
+
+export interface PersonRecord {
```

Comment:
I see what you were saying about duplication here now as this is almost a 1-1 copy and not exactly what I was thinking.

Like you are still having "nodes" and edges and such this all seems graphql specific.

I was thinking more that you'd transform the data into just the pieces you need rather than duplicating the the graphql types.

I also wonder if we shouldnt split out this embedded "works" presentation, because I'm going to need that too, therefore it would have its own types there which would just be the exact data it needs to render embedded "works" then I'd send it through from the appropriate graphql container.

We could handle this in a seperate PR though.